types/presto-client/index.d.ts

```diff
@@ -200,7 +200,18 @@ export interface RuntimeStats {
     wallTimeMillis: number;
     cpuTimeMillis: number;
     userTimeMillis: number;
-    state: "QUEUED" | "PLANNING" | "STARTING" | "RUNNING" | "FINISHED" | "CANCELED" | "FAILED";
+    state:
```

Comment:
Similar to `rawType` above, could you include a comment above this line that points at the `QueryState.java` file to make it easier to know where to look to keep this in sync?

Reply:
done